lib/matplotlib/widgets.py

```diff
@@ -3325,9 +3343,23 @@ def _release(self, event):
         self.update()
         self._active_handle = None
         self._extents_on_press = None
-
         return False
 
+    def _get_action(self):
+        """
+        Return one of "rotate", "move", "resize", "create"
```

Comment:
Let's make this an Enum instead. An enum is safer and easier to reason about compared to a bunch of strings.

Reply:
👍 done - I made the enum specific to `RectangleSelector` for now, but it's private so could always be renamed and used more generally later.